### DISCUSSION TITLE
Work with Lombiq!

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#15252] [General]
Do you like developing apps with Orchard Core? Would you like to apply your skills to some of the most challenging Orchard Core apps out there, working with other OC experts? Then come work with us at [Lombiq](https://lombiq.com/), the biggest Orchard Core team in the world!

We've been contributors, supporters, and users of Orchard Core and Orchard 1 for more than a decade (@BenedekFarkas and I, the two founders, for even more), with [a huge open-source portfolio](https://github.com/Lombiq). There you can also check out the code we write, and some of the projects you could take part in.

We do a lot. Ready for keywords? Open-source, Orchard Core CMS, .NET software development, ASP.NET Core MVC, distributed team, cloud-first, Microsoft Azure, self-funded R&D, Vue.js, jQuery, Bootstrap, SASS, training, hosting and operations, GitHub Actions, TeamCity, support, SaaS (the only Orchard CMS SaaS, DotNest: https://dotnest.com/), Selenium UI testing, xUnit, university courses. And also greenfield experimental projects like Hastlayer (https://hastlayer.com/), turning software into computer chips.

Write to us at crew@lombiq.com and let's talk!

**Comment by hishamco** [comment]
I'm interested on this to have one of the blue shirts :)

**Reply by Piedone** [reply]
You already kind of work with us, but thank you :).
